app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
There are at least a few CVEs for ruby 2.1.5p275
* https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/
* http://blog.rubygems.org/2015/05/14/CVE-2015-3900.html
* https://www.ruby-lang.org/en/news/2015/12/16/unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551/